<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Community Structure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Community Structure</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <column>
              <property name="text">
               <string>Brain region</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="topMargin">
              <number>20</number>
             </property>
             <item row="0" column="1">
              <widget class="QRadioButton" name="btnDynamic">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Dynamic structure</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="btnFixed">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Fixed structure</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="rightMargin">
                <number>10</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>10</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="leftMargin">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="btnLouvain">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Louvain algorithm</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroup_2</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="btnNewman">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Newman algorithm</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroup_2</string>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="spinBoxGamma">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.050000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelGamma">
                 <property name="text">
                  <string>Gamma</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnSet">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnReset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="BrainAtlasWidget" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFigure">
    <property name="title">
     <string>Figure</string>
    </property>
    <addaction name="actionGenerate_figure_2"/>
   </widget>
   <addaction name="menuFigure"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="action3D_rotation"/>
   <addaction name="action3D"/>
   <addaction name="actionSagittal_left"/>
   <addaction name="actionSagittal_right"/>
   <addaction name="actionAxial_dorsal"/>
   <addaction name="actionAxial_ventral"/>
   <addaction name="actionCoronal_anterior"/>
   <addaction name="actionCoronal_posterior"/>
   <addaction name="separator"/>
   <addaction name="actionBrain"/>
   <addaction name="actionShow_axis"/>
   <addaction name="actionShow_grid"/>
   <addaction name="actionShow_symbols"/>
   <addaction name="actionRegions"/>
   <addaction name="actionShow_labels"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionGenerate_figure">
   <property name="text">
    <string>Generate figure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="action3D_rotation">
   <property name="text">
    <string>3D rotation</string>
   </property>
  </action>
  <action name="action3D">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_3d.png</normaloff>:/buttons/icon_view_3d.png</iconset>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionSagittal_left">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_sl.png</normaloff>:/buttons/icon_view_sl.png</iconset>
   </property>
   <property name="text">
    <string>Sagittal left</string>
   </property>
  </action>
  <action name="actionSagittal_right">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_sr.png</normaloff>:/buttons/icon_view_sr.png</iconset>
   </property>
   <property name="text">
    <string>Sagittal right</string>
   </property>
  </action>
  <action name="actionAxial_dorsal">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_ad.png</normaloff>:/buttons/icon_view_ad.png</iconset>
   </property>
   <property name="text">
    <string>Axial dorsal</string>
   </property>
  </action>
  <action name="actionAxial_ventral">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_av.png</normaloff>:/buttons/icon_view_av.png</iconset>
   </property>
   <property name="text">
    <string>Axial ventral</string>
   </property>
  </action>
  <action name="actionCoronal_anterior">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_ca.png</normaloff>:/buttons/icon_view_ca.png</iconset>
   </property>
   <property name="text">
    <string>Coronal anterior</string>
   </property>
  </action>
  <action name="actionCoronal_posterior">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_view_cp.png</normaloff>:/buttons/icon_view_cp.png</iconset>
   </property>
   <property name="text">
    <string>Coronal posterior</string>
   </property>
  </action>
  <action name="actionBrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_brain.png</normaloff>:/buttons/icon_brain.png</iconset>
   </property>
   <property name="text">
    <string>Brain</string>
   </property>
  </action>
  <action name="actionShow_axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_axis.png</normaloff>:/buttons/icon_axis.png</iconset>
   </property>
   <property name="text">
    <string>Show axis</string>
   </property>
  </action>
  <action name="actionShow_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_grid.png</normaloff>:/buttons/icon_grid.png</iconset>
   </property>
   <property name="text">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionShow_symbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_symbol.png</normaloff>:/buttons/icon_symbol.png</iconset>
   </property>
   <property name="text">
    <string>Show symbols</string>
   </property>
  </action>
  <action name="actionRegions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_sphere.png</normaloff>:/buttons/icon_sphere.png</iconset>
   </property>
   <property name="text">
    <string>Regions</string>
   </property>
  </action>
  <action name="actionShow_labels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/icon_label.png</normaloff>:/buttons/icon_label.png</iconset>
   </property>
   <property name="text">
    <string>Show labels</string>
   </property>
  </action>
  <action name="actionGenerate_figure_2">
   <property name="text">
    <string>Generate figure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrainAtlasWidget</class>
   <extends>QGraphicsView</extends>
   <header>braphy.gui.widgets.brain_atlas_widget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
